modules/installation-gcp-service-account.adoc

```diff
@@ -30,8 +30,12 @@ See link:https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#gr
 While making the service account an owner of the project is the easiest way to gain the required permissions, it means that service account has complete control over the project. You must determine if the risk that comes from offering that power is acceptable.
 ====
 
-. Create the service account key in JSON format.
-See link:https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys[Creating service account keys]
-in the GCP documentation.
+. Either create the service account key in JSON format, or attach the service account to a GCP virtual machine.
+See link:https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys[Creating service account keys] and link:https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances[Creating and enabling service accounts for instances] in the GCP documentation.
++
+You must either have a service account key or a virtual machine with an attached service account to create the cluster.
 +
-The service account key is required to create a cluster.
+[NOTE]
+====
+If you use a virtual machine with an attached service account to create your cluster, you must set `credentialsMode: Manual` in the `install-config.yaml` file before installation.
```

Comment:
Better to tell what's needed when using `credentialsMode: Manual`, e.g. https://docs.openshift.com/container-platform/4.11/installing/installing_gcp/manually-creating-iam-gcp.html. Thanks!

Reply:
Thanks @jianli-wei , I have added a reference to the "Manually create IAM" page immediately after this note. I would prefer not to detail the steps of manual IAM when we have an entire page dedicated to that. Does this look good to you?

Reply:
/lgtm